roles/rook/README.rst

```diff
@@ -0,0 +1,113 @@
+Rook
+====
+
+This role is used for updating and recovering the rook jenkins in the rook ci vpc.
+
+Those are the functions in this role are
+
+- rook-jenkins-update For updating rook jenkins version to the version defined in the "jenkins_master_image" varible
+
+- rook-os-update For updating rook jenkins OS packages
+
+- rook-recovery For recoverying the Prod-jenkins instance from the image defined in the "image" varible in a case that the instance was deleted or crashed
+
+Usage
++++++
+
+The rook role is used by the ``rook.yml`` playbook.  Run this playbook with one of the optional tags listed in the tags section to upgrade rook jenkins OS packages/recover it from an image or update the rook jenkins app.
+
+**Pre-requisites:** Before running ``rook.yml`` make sure your ip address has ssh access to the vpc this is configured in the aws dashboard, "rook-jenkins-group" security group inbound roles
+
+- The Rook-Recovery Playbook is used for deploying rook jenkins from an image in case of a crash/corruption, How-to-run this play:
+    - Run the playbook with the rook-recovery tag, then you will need to make the newly created instance available to the public network as explained in the next step.
+
+    - Once the instance is deployed, Now add the deployed instance to the load balancing target group named "jenkins-rook-new" so that he will be available to the public network.
```

Comment:
```suggestion
- The Rook-Recovery Playbook is used for deploying rook jenkins from an image in case of a crash/corruption:
    # Run the playbook with the rook-recovery tag, then you will need to make the newly created instance available to the public network as explained in the next step.

    # Once the instance is deployed, Now add the deployed instance to the load balancing target group named "jenkins-rook-new" so that he will be available to the public network.
```